### ISSUE TITLE
Code Review - Ben

### ISSUE THREAD

**Issue by WoodsyTheNerd** [th0mazth0maz/OOPAssignmentOne#3]
### 👍Great Parts
- Quite clearly written and easy to understand
- Strong exception and error handling
- Well commented
- Included iteration allowing multiple hands to be dealt out 
- Automated re-dealing after running out of cards was a great idea

### ⚒️Suggestions 
- (personal) it may provide a more positive user experience and provide a more polished experience to only show processed cards upon request, not upon deck creation.
- Bug/exception error when a noninteger value is provided to the number of cards being dealt leading to the program freezing, creating a System.FormatException Error and having to be forcibly stopped.
![Screenshot 2023-03-08 152718](https://user-images.githubusercontent.com/119528792/223755187-8c21d3fd-3159-49c7-bc8f-d1b9e0aaafc6.png)

This could likely be rectified with some kind of integer checking mechanism like so:
int amount;
```
if (int.TryParse(n, out value))
{
<process>
}
else
{
<process>
}
```

tl;dr: Very strong code. Generally strong error handling, strong commenting but minor bugs during intentional bug testing. There may also be a slight benefit for a bit of UI polishing.
